umbraco/Umbraco-CMS
# Fixes fallback value ignored in Localize directive

### Prerequisites


### Description
While testing some of the PR's from @BatJan about adding fallback values to the localize directives as specified in the documentation, I noticed that the current implementation of the directive actually ignored the fallback value provided.

This PR fixes that and uses the initial inner HTML of the localize element as fallback value.

In order to test: 

- With the previous version of the code, open a file with localization elements and fallback values, and update one of the keys so that it won't be found in the localization files.

I took for example the file Umbraco.Web.UI.Client\src\views\mediatypes\views\permissions\permissions.html and added a "1" at the end of the first localization key.

![image](https://user-images.githubusercontent.com/1769475/142775550-48c1d937-fadb-4af5-8ff2-973b31a351e8.png)

- Check in the browser, you will see the localization key instead of the fallback text "Allow As Root"

![image](https://user-images.githubusercontent.com/1769475/142775600-1666159e-3906-4460-8ab6-da5f1ca6cf5f.png)

- Update the code to this PR and do the same test, you will see the fallback text as expected

![image](https://user-images.githubusercontent.com/1769475/142775709-da1bf1fb-f8aa-43ef-a759-5a84ed9805d8.png)
